docs/public/_redirects

```diff
@@ -61,6 +61,11 @@ https://material-ui.dev/* https://material-ui.com/:splat 301!
 /api/expansion-panel-actions/ /api/accordion-actions/ 301
 /api/expansion-panel-details/ /api/accordion-details/ 301
 /api/expansion-panel-summary/ /api/accordion-summary/ 301
+/components/grid-list/ /components/image-list/ 301
+/:userLanguage/components/grid-list/ /:userLanguage/components/image-list/ 301
+/api/grid-list/ /api/image-list/ 301
+/api/grid-list-tile/ /api/image-list-item/ 301
+/api/grid-list-tile-bar/ /api/image-list-item-bar/ 301
```

Comment:
ExpansionPanel also has these redirects, but they don't actually do anything, because the path exists:

https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing

Reply:
I guess we should remove the redirection on the API side in this case?

Reply:
You can force the rewrite (`301!`), but since the pages are there, we might as well document the old API for anyone still using it, until they migrate.

Reply:
Yeah, better to remove the /api/ redirections.